graylog2-web-interface/src/views/components/views/ViewHeader.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+
+import React, { useCallback, useMemo, useState } from 'react';
+import styled, { css } from 'styled-components';
+
+import { useStore } from 'stores/connect';
+import { ViewStore } from 'views/stores/ViewStore';
+import { Icon } from 'components/common';
+import { Row } from 'components/bootstrap';
+import ViewPropertiesModal from 'views/components/views/DashboardPropertiesModal';
+import onSaveView from 'views/logic/views/OnSaveViewAction';
+
+const StyledList = styled.dl(() => css`
+  display: flex;
+  gap: 8px;
+  margin: 0;
+  flex-wrap: wrap;
+  align-items: center;
+  dd {
+    font-style: italic;
+  }
+  dt {
+    text-transform: capitalize;
+  }
+`);
+
+const Content = styled.div(({ theme }) => css`
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: nowrap;
+  align-items: center;
+  padding-bottom: ${theme.spacings.sm}
+`);
+const EditButton = styled.div(() => css`
+  width: 25px;
```

Comment:
I do not think the width has any effect here.

Reply:
Look like without it Icon is placed a bit outside the container... Or it looks like that just for me?

Reply:
Not as far as I can tell:
<img width="63" alt="image" src="https://user-images.githubusercontent.com/46300478/191057679-5e448636-a7a7-49f1-bcc1-3c9494b46e98.png">